python/matplotlib.ipynb

```diff
@@ -729,7 +757,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.4"
+   "version": "3.9.13"
```

Comment:
Why are we using a different python version than below with stating that 3.10 is up to date?

Reply:
Which is also wrong, we have 3.11 now...

Reply:
This does not really matter, does it? It's just some meta data here.

Reply:
Yes, this is whatever python version was used to run the notebook locally. Christian should update ;)

Reply:
> Christian should update ;)
Never. I will go to even lower numbers.